komodo/cli.py

```diff
@@ -82,6 +82,9 @@ def _main(args):
     if args.download and not args.build:
         sys.exit(0)
 
+    if (Path(args.prefix) / args.release).exists() and "bleeding" not in args.release:
```

Comment:
I understood that we want to have an option that can still overwrite the release in the action that is.

Reply:
right!

Reply:
fixed.